src/Templates/src/templates/maui-mobile/Resources/Styles.xaml

```diff
@@ -0,0 +1,164 @@
+﻿<ResourceDictionary 
+    xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+    xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml">
+
+    <Color x:Key="AccentColor">#512bdf</Color>
+    <Color x:Key="PrimaryLightColor">White</Color>
+    <Color x:Key="PrimaryDarkColor">Black</Color>
+
+    <Color x:Key="SecondaryAccentColor">#5639b0</Color>
+    <Color x:Key="SecondaryLightColor">LightGray</Color>
+    <Color x:Key="SecondaryDarkColor">DarkSlateGray</Color>
+
+    <Style TargetType="Button">
+
+        <!-- BUTTON SPECIFIC PROPERTIES -->
+        <Setter Property="TextColor" Value="{AppThemeBinding Light={StaticResource PrimaryLightColor}, Dark={StaticResource AccentColor}}" />
+        <!--<Setter Property="CharacterSpacing" Value="0" />-->
+        <!--<Setter Property="FontFamily" Value="OpenSansRegular" />-->
+        <!--<Setter Property="FontSize" Value="14" />-->
+        <!--<Setter Property="TextTransform" Value="None" />-->
+        <!--<Setter Property="FontAttributes" Value="Bold" />-->
+        <!--<Setter Property="FontAutoScalingEnabled" Value="True" />-->
+        <!--<Setter Property="BorderWidth" Value="0" />-->
+        <Setter Property="BorderColor" Value="{AppThemeBinding Light={StaticResource AccentColor}, Dark={StaticResource PrimaryLightColor}}" />
+        <!--<Setter Property="CornerRadius" Value="0" />-->
+        <!--<Setter Property="Padding" Value="5" />-->
+        <!--<Setter Property="LineBreakMode" Value="WordWrap" />-->
+
+        <!-- VIEW INHERITED PROPERTIES -->
+        <!--<Setter Property="VerticalOptions" Value="Center" />-->
+        <!--<Setter Property="HorizontalOptions" Value="Center" />-->
+        <!--<Setter Property="Margin" Value="0" />-->
+
+        <!-- VISUALELEMENT INHERITED PROPERTIES -->
+        <!--<Setter Property="Shadow" Value="" />-->
+        <!--<Setter Property="Width" Value="100" />-->
+        <!--<Setter Property="Height" Value="25" />-->
+        <!--<Setter Property="Clip" Value="" />-->
+        <!--<Setter Property="IsVisible" Value="True" />-->
+        <Setter Property="BackgroundColor" Value="{AppThemeBinding Light={StaticResource AccentColor}, Dark={StaticResource PrimaryLightColor}}" />
```

Comment:
Should we standardize on Background instead of BackgroundColor so we can leverage brushes everywhere (or is it called paint now?) and easily swap gradients?

Reply:
will do!